repository/gridftp/globus5/authz/CMakeLists.txt

```diff
@@ -1 +1,5 @@
 add_subdirectory( source )
+
+if( BUILD_AUTHZ_TESTS )
+  add_subdirectory( tests )
```

Comment:
Does `add_subdirectory` tell CMake to search the subdirectory for another List?

Reply:
Exactly.